app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -522,7 +524,16 @@ class NewMessageFragment : Fragment() {
             addressListPopupWindow.dismiss()
         }
 
-        fromMailAddress.post { addressListPopupWindow.width = fromMailAddress.width }
+        fromMailAddress.post {
+            runCatching {
+                addressListPopupWindow.width = fromMailAddress.width
+            }.onFailure { exception ->
+                Sentry.withScope {
+                    Sentry.captureException(exception)
+                    Sentry.captureMessage("Binding null in post(), this is not normal", SentryLevel.WARNING)
```

Comment:
This will send 2 sentries at once.
We need to choose which one we want to send.